modules/account-lib/src/coin/near/utils.ts

```diff
@@ -73,6 +73,10 @@ export class Utils implements BaseUtils {
     return bs58.encode(value);
   }
 
+  base58Decode(value: string): Uint8Array {
```

Comment:
Will this be used for more than a test? If not, can we import bs58 into the test?